### ISSUE TITLE
Vanilla “Show Hitboxes” won't work after install this mod.

### ISSUE THREAD

**Issue by su226** [maxnut/GDmenu#359]
Vanilla “Show Hitboxes” (Settings > Options > Practice > Show Hitboxes) won't work after install this mod, even if “Show Hitboxes” of this mod is not turned on.

Same as *part of* #286. Opened a new issue because it's still not fixed. (Seems only “black hitboxes” is fixed)

> Practice mode hitboxes don't work like in the editor too.

**Comment by maxnut** [comment]
yeah, because the mod replaces the vanilla feature, use that instead

**Comment by su226** [comment]
Alright, thanks.

I just felt annoying to turn it on and off again. Maybe make it respect vanilla settings (better, because toggleable with BetterPause or other mods), or add an option in "Show hitboxes settings" like this?

![Example](https://github.com/maxnut/GDMegaOverlay/assets/17371317/a3f002b2-fa72-488a-ab94-476207ab6f94)

ps: Only "Always show" should be treated as cheat, not "Show on death", nor "Show in practice mode", of course.
pps: I wanted to use "Show otherwise"/"Show in normal mode" first, but I thought there's almost no reason to show hitboxes in normal mode but not practice mode.

**Comment by maxnut** [comment]
but always show is literally the default
